tensorflow/compiler/xla/tests/llvm_compiler_test.cc

```diff
@@ -156,8 +168,13 @@ TEST_F(CpuCompilerTest, CpuMultiModuleCompilation) {
   TestMultiModuleCompilation(&compiler);
 }
 
-TEST_F(GpuCompilerTest, NVPTXMultiModuleCompilation) {
+TEST_F(GpuCompilerTest, GpuMultiModuleCompilation) {
+#if GOOGLE_CUDA
```

Comment:
#if !TENSORFLOW_USE_ROCM . let's try avoid GOOGLE_CUDA within XLA, as there really is no CUDA within XLA but only NVPTX.